scripts/default_config.py

```diff
@@ -116,6 +116,7 @@ def get_default_config():
     cfg.train.lr = 0.0003
     cfg.train.weight_decay = 5e-4
     cfg.train.max_epoch = 60
+    cfg.train.epoch_scale = 3.
```

Comment:
This is a task-specific parameter, which doesn't affect train via main.py. Logically it doesn't belong to d-o-r yaml config. We can put the new params to ConfigurableParameters, however, they are not convenient to modify. I propose adding this stuff to yaml, into a standalone group like `sc_integration`, subgroup `multilabel`.

Reply:
done